styles/templates/types.ts

```diff
@@ -15,6 +15,7 @@ type TemplateLayout = {
 }
 
 type TemplateSpaces = [number, number, number, number]
+type TemplateSpacesBottom = [number | string, number | string]
```

Comment:
Não seria que nem a sugestão?
```suggestion
type TemplateSpacesBottom = [number, number | string, number | string, string]
```

Reply:
Não, seriam somente 2 parâmetros